src/rdkafka_queue.h

```diff
@@ -42,6 +42,13 @@
 
 TAILQ_HEAD(rd_kafka_op_tailq, rd_kafka_op_s);
 
+/**
+ * @struct Queue for rd_kafka_op_t*.
+ *
+ * @remark All readers of the queue must call rd_kafka_q_mark_served()
+ *         after reading the queue (while still holding the queue lock) to
+ *         clear the wakeup-sent flag.
```

Comment:
seems like something that should be encapsulated, but also ok everything considered

Reply:
It is encapsulated in the implementation (queue.c and queue.h)